Example/SygicUIKit/SkinTestViewController.swift

```diff
@@ -0,0 +1,105 @@
+import UIKit
+import SygicUIKit
+
+class CustomTestColorPallete: SYUIColorPalette {
+    var action: UIColor { return UIColor(red: 0.7, green: 0.13, blue: 0.18, alpha: 1) }
+}
+
+struct CustomFontFamily: SYUIFontFamily {
+    public var semiBold: String { return "Oswald-Bold" }
+}
+
+class SkinTestViewController: UIViewController {
+    
+    let customSkin = SYUINightColorPalette()
+    
+    let defaultSkinButton = SYUIActionButton()
+    let nightSkinButton = SYUIActionButton()
+    let customSkinButton = SYUIActionButton()
+    
+    var defaultFontFamily: SYUIFontFamily!
+    var customFontFamily: CustomFontFamily!
+    
+    private var poiDetailVC: SYUIPoiDetailViewController?
+    private let poiDetailDataSource = PoiDetailTestViewController()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        setupButtons()
+        
+        defaultFontFamily = SYUIFontManager.shared.currentFontFamily
+        customFontFamily = CustomFontFamily()
+        
+        updateColors()
+        NotificationCenter.default.addObserver(self, selector: #selector(updateColors), name: Notification.Name(ColorPaletteChangedNotification), object: nil)
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        
+        let poiDetailVC = SYUIPoiDetailViewController()
+        poiDetailVC.dataSource = poiDetailDataSource
+        poiDetailVC.presentPoiDetailAsChildViewController(to: self, bounce: true, completion: nil)
+        self.poiDetailVC = poiDetailVC
+    }
+    
+    // MARK: Color update
+    
+    @objc private func updateColors() {
+        view.backgroundColor = .background
+        navigationController?.navigationBar.tintColor = .action
+    }
+    
+    // MARK: Color scheme switching
+    
+    @objc private func defaultSkinButtonPressed(_ sender: UIButton) {
+        SYUIFontManager.shared.currentFontFamily = defaultFontFamily
+        
+        SYUIColorSchemeManager.shared.setDefaultColorPalettes()
+        SYUIColorSchemeManager.shared.currentColorScheme = .day
+    }
+    
+    @objc private func nightSkinButtonPressed(_ sender: UIButton) {
+        SYUIColorSchemeManager.shared.setDefaultColorPalettes()
+        SYUIColorSchemeManager.shared.currentColorScheme = .night
+    }
+    
+    @objc private func customSkinButtonPressed(_ sender: UIButton) {
+        SYUIFontManager.shared.currentFontFamily = customFontFamily
+        
+        SYUIColorSchemeManager.shared.setDefaultColorPalettes(dayColorPalette: CustomTestColorPallete(), nightColorPalette: nil)
```

Comment:
`setDefaultColorPalettes` nazov by som mozno trochu upravil? 
hlavne to slovo Set Default, lebo hovori to bez parametrov o tom ze sa nastavuje nejaka preddefinovana defaulta color paleta. Ale zrazu to vie brat argumenty kde hodis svoju custom color paletu cize potom to uz nieje uplne default, pride mi to trochu confusing